Source/WTF/wtf/HashMap.h

```diff
@@ -94,6 +95,20 @@ class HashMap final {
 public:
     HashMap() = default;
 
+    template <typename Range>
```

Comment:
Our current style guide has no space after `template`.
```suggestion
    template<typename Range>
```